### PR TITLE
Added Contributors page

### LINKED ISSUE #124: Create Contributors Page
We have a link that is supposed to lead to a contributors page in our footer, but it doesn't link to anything because the page doesn't exist.  Let's create it.

### PR DESCRIPTION
**Pull request by Ebenezer997**
Created  HTML file for the contributor page. We follow the format of the LSF contributors page. We added contributors to the default .yml file call the contributors from there.
To test, click on the student software development team link on the footer.

### CODE REVIEW

File: app/config/default.yml [hunk 1]
```diff
@@ -23,3 +23,42 @@ mail:
     default_sender: '' #FIXME: Add sender email address
     override_addr: "j5u6j9w6v1h0p3g1@bereacs.slack.com"
     reply_to_address: "j5u6j9w6v1h0p3g1@bereacs.slack.com"
+
+
+contributors:
```
Comment (BrianRamsay) [inline app/config/default.yml:28]: We should add Will Romano, and also any of the folks who worked on the Balsamiq mockups last year.

File: app/controllers/main/routes.py [hunk 2]
```diff
@@ -113,3 +114,9 @@ def RemoveRSVP():
 
     flash("Successfully unregistered for event!", "success")
     return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
+@main_bp.route('/contributors',methods = ['GET'])
+def contributors():
+    contribs = app.config['contributors']
+    return render_template("/Contributors.html",
```
Comment (BrianRamsay) [inline app/controllers/main/routes.py]: the name of this file should be lower-case, like all of our other templates

File: app/controllers/main/routes.py [hunk 3]
```diff
@@ -113,3 +114,9 @@ def RemoveRSVP():
...
+           cfg = contribs
```
Comment (BrianRamsay) [inline app/controllers/main/routes.py]: You don't need to pass this into the page. The application config is available to templates https://stackoverflow.com/questions/7104198/flask-accessing-the-config-variable-in-the-template

File: app/templates/Contributors.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% set currentPage = "contributors" %}
```
Comment (BrianRamsay) [inline app/templates/Contributors.html]: Is this variable used? It's declared after base.html is included, and not used in your app_content block.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

**Comment by Ebenezer997** [comment]
Fixed PR issuess

File: app/controllers/main/routes.py [hunk 5]
```diff
@@ -113,3 +114,9 @@ def RemoveRSVP():
 
     flash("Successfully unregistered for event!", "success")
     return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
+@main_bp.route('/contributors',methods = ['GET'])
+def contributors():
+    # contribs = app.config['contributors']
+    return render_template("/contributors.html"
+           # cfg = contribs
```
Comment (BrianRamsay) [inline app/controllers/main/routes.py]: don't just leave it commented out. It's not useful anymore, and if someone really wants to see what it used to be they can look at the history of the file.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Looks good, just one change.

**Review by BrianRamsay — APPROVED** [review]
Looks good, thanks.

**Pull request merged**